src/store/handles.js

```diff
@@ -0,0 +1,15 @@
+let { toArray, extend } = require('../mindash');
+
+function handles() {
+  let constants = toArray(arguments);
+
+  return function (target, name, descriptor) {
+    target.handlers = extend({}, target.handlers, {
```

Comment:
Cool - no need to muck with `hasOwnProperty` or whatever in this case like for `Application#register`!

At some point later on it might be possible to use this to hide the handler, though it'd require changing how `Store#handlers` looks.